config/locales/en.yml

```diff
@@ -1644,6 +1644,42 @@ en:
       reset: "Reset Password"
       flash changed: "Your password has been changed."
       flash token bad: "Did not find that token, check the URL maybe?"
+  preferences:
+    show:
+      title: My Preferences
```

Comment:
This, and the other titles are in title case but I think we generally only capitalise the first word of page titles? Though I'm not sure I don't prefer it this way...

Reply:
Oh, I've never noticed. We seem to be inconsistent:

* Initial - GPS traces, Users' diaries, My settings
* Title case - Copyright and License, Getting Help, My Messages

So I think we should keep the title case here and converge on that in the future.